packages/components/src/tooltip/test/index.tsx

```diff
@@ -0,0 +1,278 @@
+/**
+ * External dependencies
+ */
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * WordPress dependencies
+ */
+import { shortcutAriaLabel } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../button';
+import Modal from '../../modal';
+import Tooltip, { TOOLTIP_DELAY } from '..';
+import cleanupTooltip from './utils/';
+
+const props = {
+	children: <Button>Button</Button>,
+	text: 'tooltip text',
+	delay: TOOLTIP_DELAY,
```

Comment:
Not exactly sure why we need this here given that `TOOLTIP_DELAY` is the default.

Reply:
Good catch! I may have added that in when testing, as I often had the delay off, so it was quicker to test.